R/covid_flag.R

```diff
@@ -2,34 +2,24 @@
 #' COVID Flag
 #'
 #' @description
-#' `covid_flag` returns COVID-19 diagnoses status
-#' for hospital admissions based on ICD-10-CA
+#' `covid_flag` returns COVID-19 diagnosis status
+#' for each `genc_id` based on ICD-10-CA diagnosis codes.
 #'
-#' Return a dataframe with two separate flags identifying
+#' Returns a table with two separate flags identifying
 #' encounters with confirmed and suspected COVID-19 diagnoses
-#' using ICD-10-CA U071 and U072 respectively
+#' using ICD-10-CA U07.1 and U07.2 respectively.
 #'
-#' A confirmed COVID-19 diagnosis (U071) is coded when
-#' there is a positive COVID-19 test regardless of the the test type.
+#' A confirmed COVID-19 diagnosis (U07.1) is coded when
+#' there is a positive COVID-19 test, regardless of the the test type.
```

Comment:
What does "regardless of the test type" refer to exactly? 

U071 is initially coded only when it's confirmed by PCR lab test. Only recently that this has been changed to include additional tests (e.g. rapid tests). So this sentence may not reflect the coding practice. I suggest rephrasing it with additional details, or remove this part.

Here is the [CIHI reference](https://www.cihi.ca/sites/default/files/document/mapping-covid-19-codes-icd-10-ca-to-base-icd-10-job-aid-en.pdf ) for change in coding practice for test type: "Change in direction, effective April 1, 2023 — U07.1 COVID-19, virus identified is assigned when COVID-19 is confirmed by a COVID-19 test result. A COVID-19 test may include a polymerase chain reaction (PCR) test, a rapid antigen test (RAT) or a point-of-care molecular test (POCT). Prior to April 1, 2023, U07.1 was assigned only when COVID-19 was confirmed by a COVID-19 lab test."

Reply:
Thanks @gemini-wenb for pointing this out! I've updated the documentation to provide the additional clarification based on the CIHI reference.